service/test/_registry_test.js

```diff
@@ -796,9 +817,22 @@ describe('Registry', function() {
       <stringified array supplied by the argument devices annotated with importMode property and deviceId property replaced by id>
       ```
       ]*/
+      /* Tests_SRS_NODE_IOTHUB_REGISTRY_06_029: [** A device information with an authentication object that contains a `type` property is considered normalized.] */
+      /* Tests_SRS_NODE_IOTHUB_REGISTRY_06_031: [A device information with an authentication object that doesn't contain the x509Thumbprint property will be normalized with a `type` property with value "sas".] */
+      /* Tests_SRS_NODE_IOTHUB_REGISTRY_06_030: [A device information with an authentication object that contains the x509Thumbprint property will be normalized with a `type` property with value "selfSigned".] */
       it('constructs a valid HTTP request', function(testCallback) {
-        var addedDevices = [{deviceId: 'deviceone'},{deviceId: 'devicetwo'}];
-        var sentBody = [{id:'deviceone',importMode:'create'},{id:'devicetwo',importMode:'create'}];
+        var addedDevices = [
```

Comment:
First is just a code formatting issue - my comment is based on code structure, i'm sure you can find a creative and readable way to format this.

Second and third don't invalidate the fact that we have unnecessary tight coupling of array content - and that it will require more effort for the maintainer to understand what's going on if you don't make that change.

So please make that change.